tests/incremental_mode/run.sh

```diff
@@ -168,7 +169,8 @@ function run() {
 	check_port_offline $WORKER1_PORT 20
 	check_port_offline $WORKER2_PORT 20
 
-	export GO_FAILPOINTS="github.com/pingcap/dm/syncer/WaitUserCancel=return(8)"
+	# only mock pull binlog failed once
+	export GO_FAILPOINTS="github.com/pingcap/dm/syncer/WaitUserCancel=return(8);github.com/pingcap/dm/syncer/GetEventError=1*return"
```

Comment:
Can you please tell me what the usage of `1 * retrun` ?

Reply:
Only trigger this error once. See: https://github.com/pingcap/failpoint#key-concepts